XCUITests/FxScreenGraph.swift

```diff
@@ -465,7 +465,21 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
 
     map.addScreenState(HistoryRecentlyClosed) { screenState in
         screenState.dismissOnUse = true
-        screenState.tap(app.buttons["Done"], to: HomePanelsScreen)
+
+        let origin = app.coordinate(withNormalizedOffset: CGVector.zero)
+        var goBackHistoryButtonCoordinate:XCUICoordinate? = nil
+
+        if isTablet {
+            screenState.gesture(to: HomePanel_History) {
+            goBackHistoryButtonCoordinate = origin.withOffset(CGVector(dx: 114.0, dy: 252.0))
```

Comment:
I guess hardcode of the coordinate is only option in this case?  if so, i guess it's okay, I wonder whether this would work for all tablets.  IF not, we should mention in comment which devices this should run under

Reply:
@npark-mozilla There is another way...recording the test it gives the button as:
`app.children(matching: .window).element(boundBy: 0).children(matching: .other).element.children(matching: .other).element.children(matching: .other).element.children(matching: .other).element.children(matching: .other).element.children(matching: .other).element(boundBy: 0).children(matching: .other).element.children(matching: .other).element.children(matching: .other).element.children(matching: .other).element.children(matching: .button).element`
but I think we will have the same issue accross sims. 
We could use the coordinates with the comment as you suggest and in the meantime ask for an ID or label for that button to make this easier if you agree

Reply:
@isabelrios sounds good, having an acc. id. would be a way to go in the long run, let's use this solution for now.